packages/finch-graphql-devtools/app/scripts/components/MessageViewer/graphql.js

```diff
@@ -0,0 +1,24 @@
+import gql from 'graphql-tag'
+
+export const MessagePullQueryDoc = gql`
+  query getMessages {
+    _finchDevtools {
```

Comment:
underscores signify that this is an internal tool?

Reply:
yep, to not take up namespace on people graph that could potentially overlap.